app/views/layouts/_messages.html.erb

```diff
@@ -1,5 +1,5 @@
 <% flash.each do |name, msg| %>
-  <% if msg.is_a?(String) %>
+  <% if msg.is_a?(String) && ![:timedout].include?(namei.to_sym) %>
```

Comment:
We could always refactor to `somelist.include?` if it becomes multiple items, but with just 1, it just requires too much thinking, IMO. My recommendation:

```ruby
namei.to_sym != :timedout
```